src/components/UI/FormSelector.vue

```diff
@@ -51,7 +51,7 @@ const id = randomString()
 <style lang="scss" module>
 .label {
   @include color-ui-secondary;
```

Comment:
ここと、FormInputのラベルの色が変わってたっぽいので直してほしいです:pray:
```suggestion
  @include color-ui-primary;
```

Reply:
ご指摘ありがとうございます。
修正しました。